cves/CVE-2004-0488.yml

```
@@ -48,12 +52,12 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: d8f71ba6b7c20cad59faa66c1a9bc93f947e651f
-     note:
+     note: modules/ssl/ssl_engine_kernel.c (ssl_hook_UserCheck): Fix buffer
```

```suggestion
     note: modules/ssl/ssl_engine_kernel.c (ssl_hook_UserCheck) Fix buffer
```
The use of a colon in YAML indicates the start of a new element, and removing this from the line will allow the Travis CI build to pass

---

Colon has been removed